.circleci/config.yml

```diff
@@ -166,6 +166,7 @@ jobs:
           at: .
       - run: unzip ./persist/dist.zip
       - run: yarn install --cwd ./apps/remix-ide-e2e --modules-folder ../../node_modules
+      - run: mkdir node_modules/hardhat && wget https://unpkg.com/hardhat/console.sol -O node_modules/hardhat/console.sol
```

Comment:
why do you need to do this?

Reply:
the console.sol is slow to load from unpkg in e2e. it caused issues

Reply:
this puts it where the localhost can find is using the nx_npm_url param